documentation/docs/getting_started/docker.md

```diff
@@ -4,20 +4,117 @@ keywords:
 
-The easiest way to start Chronicle is by using our supplied Dockerfile.
+Once you've cloned the repository from [Github](https://github.com/iotaledger/inx-chronicle.git), switch to the `docker` folder:
+```sh
+cd inx-chronicle/docker
+```
+
+## 1. Create the necessary directories for all `Hornet` and `Chronicle` databases
+
+```sh
+sudo ./create_dirs.sh
+```
+
+## 2. Configure `Hornet` by editing the `docker-compose.yml` file
+
+```sh
+<your-favorite-text-editor> docker-compose.yml
+```
+
+### Head to `Hornet`s CLI section:
+
+```yml
+hornet:
+    command:
+```
+
+### (Optional) Select another network (default=Shimmer) by uncommenting one of the `config*.json` lines as shown below:
+
+```yml
+- "-c"
+# - "config_testnet.json"
+# - "config_alphanet.json"
+```
+### Configure peers for Hornet:
+
+The most convenient way is to use autopeering. Hornet will try to find its peers automatically and learn about more and more nodes in the particular network, and switch between them regularly. 
+
+```yml
+- "--p2p.autopeering.enabled=true"
+```
+However, this also reveals the existence of your node to everyone else in the network, which you may not want to. In that case, you should set up manual peering only (but you can also use it additionally). This is how you set it up:
+
+```yml
+--p2p.peerAliases "alice","bob"
+--p2p.peers "<multiaddr_1>","<multiaddr_2>"
+```
+
+Note, that `alice` belongs to `multiaddr_1` and `bob` belongs to `multiaddr_2`. There is always a 1:1 relationship between the alias and the multiaddress, and the same order must be kept for Hornet to associate them correctly. If you don't know how a multiaddr looks like here's an example based off of [libp2p](https://docs.libp2p.io/concepts/fundamentals/addressing/).
+```
+/ip4/198.51.100.0/tcp/15600/p2p/QmYyQSo1c1Ym7orWxLYvCrM2EmxFTANf8wXmmE7DWjhx5N
+```
+
+
+Once you've set up peering, save and close the editor. There's only one final step to do.
+
+## 3. Run all containers
+
+### Running In Production
 
-First you need to setup the correct permissions for the node's database:
+Chronicle will perform best if you run it in production mode as the build will be maximally optimized. However, this comes at the cost of longer compilation times.
 
 ```sh
-./docker/prepare_docker.sh
+docker compose -f "docker-compose.prod.yml" up -d
 ```
 
-After that, with Docker installed on your system, you can spin up Chronicle by running the following command from the root of the repository.
+If you want to generate live metrics and run a Grafana server to monitor what's going on in the network, run instead:
 
 ```sh
-docker-compose -f docker/docker-compose.yml up --build
+docker compose -f "docker-compose.prod.yml" --profile "metrics" up -d
 ```
+
+Both of those will run all containers as services detached from the current terminal session, which keeps them running until you explicitly stop some or all containers with
```

Comment:
We're including a lot of general instruction on how to use docker. Again, we should simply refer to their docs.

Reply:
I agree (I know where you're coming from) AND I don't agree (I look through the glasses of a former community member myself). So ... I deliberately wrote it this way to make it as low effort as possible for the interested community to run a Chronicle and see the benefits of it.

Reply:
you're right